lib/serve/rest-api/src/utils/generate_litellm_config.py

```diff
@@ -22,6 +22,7 @@
 import yaml
 
 ssm_client = boto3.client("ssm", region_name=os.environ["AWS_REGION"])
+secrets_client = boto3.client("secretsmanager", region_name=os.environ["AWS_REGION"])
```

Comment:
Do we need to update any architecture call outs that we are taking on a new dependency for secretsmanager... Or did we already use this before?

Reply:
We already use secrets manager in the RAG operations where we need to use the postgres db there too. We've used it before 👍